srcpkgs/font-ibm-plex-ttf/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'font-ibm-plex-ttf'
 pkgname=font-ibm-plex-ttf
-version=5.1.0
+version=5.1.3
 revision=1
 archs=noarch
```

Comment:
Please remove `archs=noarch`, support for noarch packages is deprecated and will be removed shortly.